src/uct/ib/mlx5/ib_mlx5_log.c

```diff
@@ -239,6 +239,7 @@ static unsigned uct_ib_mlx5_parse_dseg(void **dseg_p, unsigned op_flags,
     return ds;
 }
 
+#if HAVE_MLX5_MMO
 static void* uct_ib_mlx5_dump_dma_seg(uct_ib_mlx5_dma_seg_t *dma_seg,
```

Comment:
need to wrap `uct_ib_mlx5_dump_dma_seg` calls with `HAVE_MLX5_MMO` too, or do we have a way to put #if/#end inside `uct_ib_mlx5_dump_dma_seg`?

Reply:
yeah, already caught and fixed